src/command.ts

```diff
@@ -96,7 +96,16 @@ export default abstract class Command {
 
   static parserOptions = {}
 
-  static enableJsonFlag = false
+  static _enableJsonFlag = false
+
+  static get enableJsonFlag(): boolean {
+    return this._enableJsonFlag
+  }
+
+  static set enableJsonFlag(value: boolean) {
+    this._enableJsonFlag = value
+    this.globalFlags = jsonFlag
```

Comment:
Shouldn't this only happen if `enableJsonFlag` is set to `true`?

Reply:
Good catch. Thanks!